docs/develop/go/test-suites.mdx

```diff
@@ -160,6 +161,103 @@ When an Activity sends a Heartbeat, be sure that you can see the Heartbeats in y
 
 If an Activity is supposed to react to a Cancellation, you can test whether it reacts correctly by canceling it.
 
+## Mock and override Nexus operations
+
+A workflow that executes a Nexus operation can be tested by providing a mock of the Nexus operation
+or overriding its implementation with the Nexus mock environment. This allows you to test the
+workflow without needing to create a Nexus handler service to run the Nexus operation.
```

Comment:
Consider switching these two sentences, along the lines of: "Mocking lets you test a Workflow without needing a Nexus handler service to run an actual Nexus operation. You can mock the Nexus operation or override its implementation with the Nexus mock environment." It establishes the motivation first and the actions second.